static/src/javascripts/projects/common/views/experiments/measure-understanding.html

```diff
@@ -0,0 +1,12 @@
+<div class="understanding">
+    <div class="understanding__text">Do you feel better informed as a result of reading this piece?</div>
+    <button class="understanding__choice js-button" data-value="yes">
```

Comment:
Yeah, it would solve it, I guess. I think radioboxes would still be more semantical, but I can see all the work behind it, to make it work properly without a submit button and stuff ... Let's go with `tabindex` 👍